packages/active-campaign-client/src/helpers/fetchSubscribedWebinarsFromDynamo.ts

```diff
@@ -0,0 +1,30 @@
+import { DynamoDBClient, QueryCommand } from '@aws-sdk/client-dynamodb';
+import { APIGatewayProxyResult } from 'aws-lambda';
+
+export async function fetchSubscribedWebinarsFromDynamo(
```

Comment:
What about `fetchWebinarSubscriptions` or `get...`? Does `fromDynamo` suffix is relevant?